modules/nw-egress-ips-about.adoc

```diff
@@ -149,6 +149,33 @@ On Azure, the following capacity limits exist for IP address assignment:
 
 For more information, see link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits?toc=/azure/virtual-network/toc.json#networking-limits[Networking limits].
 
+[id="nw-egress-ips-multi-nic-considerations_{context}"]
+== Considerations for assigning non-OVN managed networks to egress IP addresses
+
+Egress IP addresses can be set within _non-OVN managed networks_, or networks that are not managed by Open Virtual Network (OVN). By using non-OVN managed networks, {product-title} administrators are provided with a greater level of control over networking aspects, such as routing, addressing, segmentation, and security policies. Non-OVN managed networks also provide users with the option to route workload traffic over specific network interface cards (NICs) for purposes such as traffic segmentation or meeting specialized requirements.
+
+For users who want to assign non-OVN managed networks to their egress IP addresses, the following conditions must be met:
+
+* {product-title} is installed on a bare metal cluster. Assigning non-OVN managed networks to egress IP addresses is disabled in cloud and hypervisor environments, and does not work for host-networked pods. 
+* A standard Linux network link is required. If a network link is removed or if the IP address on the link is removed, then egress IP is unwired and reassigned to another location. 
+* Egress IP addresses must have a universal, or _global_, scope. This means that in order to assign an egress IP address to a network link, the system examines all IP addresses with global scopes and their associated subnets. For example, if an egress IP falls within a specific `/24` subnet such as `1.1.1.20/24`, it can be assigned to the corresponding link. The scope of a link can be determined using standard Linux tools, and you also have the ability to set the scope of an address on a link using these tools. For more information about address scopes and values, see link:https://man7.org/linux/man-pages/man8/ip-address.8.html[Linux manual page] on IP addresses.
```

Comment:
We'd talked about this some time ago, and I think it was decided that generally we're using non-routable addresses for examples. This address is owned by Cloudflare.